pkg/plugins/policies/meshtcproute/api/v1alpha1/meshtcproute.go

```diff
@@ -58,6 +58,5 @@ type Rule struct {
 }
 
 type RuleConf struct {
-	// +kubebuilder:validation:MinItems=1
-	BackendRefs []common_api.BackendRef `json:"backendRefs"`
+	BackendRefs *[]common_api.BackendRef `json:"backendRefs,omitempty"`
```

Comment:
let's validate this on in `validator.go` to not allow `rule: {}`

Reply:
done https://github.com/kumahq/kuma/pull/12844/commits/ae138ef2dc4538a41592dded739e361eac252c4a